presto-orc/src/main/java/com/facebook/presto/orc/CachingOrcDataSource.java

```diff
@@ -73,6 +75,7 @@ void readCacheAt(long offset)
         cacheLength = newCacheRange.getLength();
         if (cache.length < cacheLength) {
             cache = new byte[cacheLength];
+            systemMemoryContext.setBytes(cacheLength);
```

Comment:
is tinyStripeThreshold 8MB ? 
How about set cache size as maximal possible stripe size (8MB at most for tiny stripe?)
If memory accounting is only off by 8MB, it is probably ok. Is creating new cache too fast caused GC ?

Reply:
@viczhang861 The cacheLength is the min of tinyStripeThreshold and the size of the file. We don't want to make it always 8MB because the files may well be smaller than that. This 8MB is not small because the cache is allocated for each PageSource(typically a file) in each OrcRecordReader. Considering worker parallelism and query parallelism, this amount could be quite high.